test/e2e/snaps/test-snap-bip-32.spec.js

```diff
@@ -0,0 +1,170 @@
+const { strict: assert } = require('assert');
+const { withFixtures } = require('../helpers');
+const { TEST_SNAPS_WEBSITE_URL } = require('./enums');
+
+describe('Test Snap bip-32', function () {
+  it('tests various functions of bip-32', async function () {
+    const ganacheOptions = {
+      accounts: [
+        {
+          secretKey:
+            '0x7C9529A67102755B7E6102D6D950AC5D5863C98713805CEC576B945B15B71EAC',
+          balance: 25000000000000000000,
```

Comment:
Can this be a string or `bigint`? It's not a safe integer, and may result in unexpected behaviour.

Reply:
This is used in all the other E2Es, we can do a separate PR to fix this if you don't mind?